corehq/apps/export/templates/export/datasource_export_view.html

```diff
@@ -0,0 +1,27 @@
+{%  extends 'hqwebapp/bootstrap3/base_section.html' %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+{% load compress %}
+{% load hq_shared_tags %}
+
+{% requirejs_main 'export/js/datasource_export' %}
+
+{% block page_title %}
+{% trans "Export Data Source Data" %}
+{% endblock %}
+
+{% block page_content %}
+  <p>
+    {% blocktrans %}
+    Generate a DET query file from any of the project's data sources.
+    {% endblocktrans %}
+  </p>
+  <div id="datasources_export_id" class="form form-horizontal">
+    <div data-bind='hidden: haveDatasources'>
+      {% blocktrans %}
+      The project has no datasources configured.
```

Comment:
In other parts of the codebase, "data source" is spelled as two words. e.g. 
![image](https://github.com/dimagi/commcare-hq/assets/708421/eae50720-2b48-4ff3-8392-0d8d27ac84ec)

We should use the same spelling here.